src/bin/list_store.rs

```diff
@@ -0,0 +1,299 @@
+// "List Store" gtk3 demo ported to gtk-rs by Ömer Sinan Ağacan <omeragacan@gmail.com>
+
+extern crate gio;
+extern crate gtk;
+
+use gio::*;
+use gtk::*;
+
+use std::env::args;
+use std::rc::Rc;
+
+fn main() {
+    let application =
+        gtk::Application::new("com.github.gtk-rs.examples.list-store", Default::default())
+            .expect("Initialization failed...");
+
+    application.connect_activate(|app| {
+        build_ui(app);
+    });
+
+    application.run(&args().collect::<Vec<_>>());
+}
+
+static COLUMN_FIXED: i32 = 0;
```

Comment:
Rust has enums you know? :)

Reply:
I don't need enums though, I need `i32`s. Can I convert a Rust enum variant to `i32`?

Reply:
Yep: `#[repr(i32)]`.